composer.json

```
@@ -25,18 +25,21 @@
 		"psalm:update-baseline": "psalm.phar --threads=1 --update-baseline",
 		"psalm:clear": "psalm.phar --clear-cache && psalm --clear-global-cache",
 		"psalm:fix": "psalm.phar --alter --issues=InvalidReturnType,InvalidNullableReturnType,MissingParamType,InvalidFalsableReturnType",
```

Does this work? as `vimeo/psalm` is used I would expect it needs to be this now:
```suggestion
		"psalm:fix": "psalm --alter --issues=InvalidReturnType,InvalidNullableReturnType,MissingParamType,InvalidFalsableReturnType",
```